doc/distributed.md

```diff
@@ -25,23 +25,25 @@ docker run --rm -d --name=graphite --network amoc-test-network \
 Start two Amoc containers, export all of the necessary environmental variables so that the nodes can communicate with each other and send metrics to Graphite.
 In order to use Amoc HTTP API for uploading and starting scenarios, port 4000 should be published.
 ```
-docker run --rm -t -d --name amoc-1 -h amoc-1 --network ${NETWORK} \
+docker run --rm -t -d --name amoc-1 -h amoc-1 \
+    --network amoc-test-network \
     -e AMOC_HOSTS="\"amoc-1\",\"amoc-2\"" \
     -e AMOC_GRAPHITE_HOST=graphite \
     -e AMOC_GRAPHITE_PORT=2003 \
     -e AMOC_PREFIX=amoc1 \
-    --health-cmd="${PATH_TO_AMOC} status" \
+    --health-cmd="/home/amoc/amoc/bin/amoc status" \
     -p 8081:4000 \
-    amoc-reworked:latest
+    amoc_image:tag
```

Comment:
I think it'd be better to put a real amoc image tag like `mongooseim/amoc:latest`. People probably will want to copy and paste this command to start amoc node.

Reply:
actually, `tag` is a valid tag. you can copy-past that. i did it for testing :)

Reply:
How about `amoc_image:tag` this only works if you have it locally. If you don't then docker will attempt do download it from dockerhub and there, there is no `amoc_image:tag`.

Reply:
Ah, now I see. The image is build in a step at the very beginning of the doc page. No need to change that.

Reply:
and the first instruction in the file :
```
docker build -f Dockerfile -t amoc_image:tag .
```
this PR is not about documentation, i just made instructions in the file compatible and copy-pastable.